package.json

```diff
@@ -16,13 +16,15 @@
   },
   "author": "",
   "license": "MIT",
+  "dependencies": {
+    "ember-cli-htmlbars": "0.7.4"
+  },
   "devDependencies": {
-    "broccoli-asset-rev": "^1.0.0",
-    "broccoli-ember-hbs-template-compiler": "^1.6.1",
-    "ember-cli": "0.1.4",
-    "ember-cli-dependency-checker": "0.0.6",
+    "broccoli-asset-rev": "^2.0.0",
+    "ember-cli": "0.2.0",
+    "ember-cli-dependency-checker": "0.0.7",
     "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-qunit": "0.1.2",
+    "ember-cli-qunit": "0.3.9",
```

Comment:
I bet we can get away with a `^0.3.0` here. `ember-cli-qunit` doesn't seem like the kind of dependency that is going to cause a mudslide.